docs/business-apps/power-automate/get-started/create-your first-flow.md

```diff
@@ -49,16 +46,16 @@ Before proceeding with these instructions, make sure your SharePoint site is set
 
     ![Flow template - send email when an item is added in a list](../../../images/gs01-create-a-flow-when-item-is-added-template.png)
 
-6. To create the flow, select **Create Flow**. Any connections that require a connection is also attempted during this step. After the flow is created successfully, you are redirected to the flow designer where you can edit and modifify the flow if needed.
+6. To create the flow, select **Create Flow**. Any connections that require a connection is also attempted during this step. After the flow is created successfully, you are redirected to the flow details page where you can edit and modifify the flow if needed.
```

Comment:
Any connections that require a connection are also attempted during this step. (replace 'is' with 'are.') After you successfully create the flow, the flow details page appears where you can edit and modify the flow, if needed. (rebuild sentence and fix 'modify' misspelling).